### PR TITLE
Add instructions to setup APPUiO Keycloak extensions

### PR DESCRIPTION
**Pull request by ccremer**
* Adds a how-to page to install appuio-keycloak-extensions at startup

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

**Comment by ccremer** [comment]
This PR is waiting on https://github.com/projectsyn/component-keycloak/pull/67

File: docs/modules/ROOT/pages/how-to/keycloak-brokering.adoc [hunk 1]
```diff
@@ -58,6 +58,19 @@ Direct Access Grants Enabled = True
...
+  Go to the "Mappers" tab and create a new Mapper.
++
+[source,subs="attributes+"]
+----
+Name = group membership
+Mapper type = Group Membership
+Token Claim Name = groups
+Full group path = false <1>
+----
+<1> If this would be `true` (default), then a group name like `/Company/admins` would contain group `/Company/admins`, which might be undesirable.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/keycloak-brokering.adoc:71]: This sentence is odd, I think you wanted to say something like:

```suggestion
<1> If this is `true` (the default), then the group name for group `/Company/admins` is `/Company/admins`, which might be undesirable.
```

File: docs/modules/ROOT/pages/how-to/keycloak-brokering.adoc [hunk 2]
```diff
@@ -58,6 +58,19 @@ Direct Access Grants Enabled = True
...
+    If `false`, the group name would be just the group name of the leaf node, for example `admins`.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/keycloak-brokering.adoc:72]:
```suggestion
    If this is `false`, the group name is just the value of the leaf node instead, for example group `/Company/admins` will be assigned the name `admins`.
```

**Review by simu — APPROVED** [review]
Looks good overall, some suggestions regarding the explanations for the `Full group path` parametr

**Pull request merged**